src/vivarium_gates_mncnh/components/mortality.py

```diff
@@ -163,3 +164,79 @@ def get_proportional_case_fatality_rates(self, simulants: pd.DataFrame) -> pd.Da
             )
 
         return simulants
+
+
+class NeonatalMortality(Component):
+    """A component to handle neonatal mortality."""
+
+    ##############
+    # Properties #
+    ##############
+
+    @property
+    def configuration_defaults(self) -> dict[str, Any]:
+        return {
+            self.name: {
+                "data_sources": {
+                    "all_cause_mortality_rate": "cause.all_causes.cause_specific_mortality_rate",
+                    "life_expectancy": "population.theoretical_minimum_risk_life_expectancy",
```

Comment:
Both mortality components (for mothers and now children) have this same lookup table but that seems fine since the lookup tables are tied to the components.